src/jquery.uls.lcd.js

```diff
@@ -419,6 +503,18 @@
 		listen: function () {
 			var lcd = this;
 
+			this.$element.on( 'mouseenter', '.row li', function () {
+				lcd.navigationIndex = $( this ).index();
+				lcd.highlightLanguageOption();
+			} );
+
+			this.$element.on( 'mouseleave', '.row li', function () {
+				if ( lcd.navigationIndex === $( this ).index() ) {
```

Comment:
When pressing the down arrow if the language being highlighted is not visible, the ULS should scroll down to display the currently highlighted language item.

Reply:
Fixed in 8e0a9797365927f8936d010d2cdb8737b312b791